lib/usd/ui/layerEditor/layerTreeItem.cpp

```diff
@@ -10,6 +10,7 @@
 #include "warningDialogs.h"
 
 #include <mayaUsd/base/tokens.h>
+#include <mayaUsd/fileio/jobs/jobArgs.h>
```

Comment:
Is this include necessary?

Reply:
This is no longer necessary as the function now lives in utilSerialization. I'll remove it.